resume/src/app/experience-card/experience-card.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="shadow rounded-xl p-7 block-section bg-white mb-5">
+  <h2 class="text-xl font-semibold mb-1.5">Experience</h2>
+
+  <div class="mb-5 flex items-start">
+    <div class="company-logo mx-2">
+      <img src="https://ipt.ch/static/img/logo_ipt.e41536c42e31.svg" alt="Avatar"
+           class="w-[40px] border-dark-blue rounded-full">
+    </div>
+
+    <div class="w-full space-y-5">
+      <div class="flex justify-between">
+        <div class="space-y-1.5">
+          <div class="font-medium">Consultant</div>
+          <div class="flex space-x-5">
+            <div class="flex items-center">
+              <svg xmlns="http://www.w3.org/2000/svg" class="h-4 w-4 mr-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Comment:
Do you have a good source for svgs? I always used mat-icons and struggled a bit to find the svgs I wanted